.github/workflows/dockerbuild.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - uses: crazy-max/ghaction-docker-buildx@v1
         if: ${{ github.event_name != 'pull_request' }}
 
-      - name: Latest
+      - name: Dev
         if: ${{ github.ref == 'git/refs/heads/master'}}
```

Comment:
Is this if needed as dev tag would always be builded?

Reply:
i'm relying on the build cache, so it'll fly through the second one. it's not elegant but docs on buildx aren't great :(